### DISCUSSION TITLE
How to know if Keycloak 21.1.1 is running on FIPS mode

### DISCUSSION THREAD

**Discussion by kabipatt** [keycloak/keycloak#22208] [Archive]
Hello,
I am running Keycloak 21.1.1 in a containerized environment.
I have added the [recommended](https://www.keycloak.org/server/fips) bouncy castle jar files as providers and also using a BCKFS keystore with following env variables.

```
ENV KC_FEATURES=fips,token-exchange
ENV KC_FIPS_MODE=strict
ENV KC_HTTPS_KEY_STORE_TYPE=BCFKS
:
and so on
```
The server is starting fine, but there no sign of BCFIPS running on "approved mode" in the logs. I suppose to see this line in the log as per this [documentation](https://www.keycloak.org/server/fips). But I don't see it. 

> KC(BCFIPS version 1.000203 Approved Mode, FIPS-JVM: enabled) version 1.0 - class org.keycloak.crypto.fips.KeycloakFipsSecurityProvider,

How will I know it is truly running on strick FIPS approved mode?

**Comment by trixpan** [comment]
The log is in TRACE mode.

@mposolda I suspect either the log needs to moved from TRACE or the documentation updated?
